testing/integration/upgrade_fleet_test.go

```diff
@@ -215,6 +259,7 @@ func testUpgradeFleetManagedElasticAgent(
 			URL:             fleetServerURL,
 			EnrollmentToken: enrollmentToken.APIKey,
 		},
+		Unprivileged: atesting.NewBool(unprivileged),
```

Comment:
In the `t.Log("Installing Elastic Agent...")` statement above, can you make this include the priviledge flag? Like `t.Logf("Installing Elastic Agent (unprivileged: %t)...", unprivileged)`

Reply:
Added.